wagtail/contrib/search_promotions/templates/wagtailsearchpromotions/includes/searchpromotion_form.html

```diff
@@ -22,7 +22,23 @@
             </ul>
         {% endif %}
 
-        {% include "wagtailadmin/shared/field.html" with field=form.page only %}
+        <p>&nbsp;</p>
+        <strong>
+            {% trans "Recommend either a" %}
+        </strong>
+        <div class="nice-padding indent-10">
+            {% include "wagtailadmin/shared/field.html" with field=form.page only %}
+        </div>
+
+        <strong>
+            {% trans "Or an external link" %}
+        </strong>
```

Comment:
We may need to revisit this content, breaking up the translation parts like this will be difficult to work with for some languages. 

Plus, this content is likely going to need some refinement for accessibility. 

I'll run this locally and try to give some suggestions. 

For now - we should at least remove the empty paragraph with the space in it. CSS should be used if we need to do layout things.

Reply:
OK here is a recommended set of HTML for this area, the wording is a bit verbose but I think it may be a better place to start. Essentially using simpler semantic HTML basics (fieldset/legend) to break up the form. This also will work much better for screen readers.

I will reach out to the rest of the core team for any additional guidance on wording here but hopefully this will be a step in the right direction.

```html
    {% panel id=panel_id heading=heading heading_size="label" header_controls=header_controls %}
        {% if form.non_field_errors %}
            <ul>
                {% for error in form.non_field_errors %}
                    <li class="error-message">
                        <span>{{ error|escape }}</span>
                    </li>
                {% endfor %}
            </ul>
        {% endif %}
        <fieldset>
            <legend>{% trans "Use an internal Page for this Promotion" %}</legend>
            {% include "wagtailadmin/shared/field.html" with field=form.page only %}
        </fieldset>
        <fieldset>
            <legend>{% trans "Alternatively, use an external link for this Promotion" %}</legend>
            {% include "wagtailadmin/shared/field.html" with field=form.external_link only %}
            {% include "wagtailadmin/shared/field.html" with field=form.external_title only %}
        </fieldset>
        {% include "wagtailadmin/shared/field.html" with field=form.description only %}
    {% endpanel %}
```

<img width="1958" alt="Screenshot 2023-08-24 at 7 56 35 am" src="https://github.com/wagtail/wagtail/assets/1396140/acb0eb1f-ff10-44ba-99a4-dd9bc3605ead">